cypress/test/search-column-selector.spec.ts

```diff
@@ -49,6 +47,7 @@ describe('Onto Search Column Selector Component', () => {
     step.getColumnSelectDropdown().click();
     // Only 'Name' dropdown option should be selected
     step.getSelectedOptions().should('have.length', 1).and('contain', 'Name');
+    step.closeColumnSelectDropdown();
     // And the selected columns should be the one passed to component ('Name')
     step.getSelectedColumns().should('contain', 'Name');
```

Comment:
I ususally try to close menus at the end of the tests. Not a big deal but it might cause troubles in some rare occasions

Reply:
I close it here because the selection is emitted on panel closed.